test/conc/plus.wat

```diff
@@ -0,0 +1,8 @@
+(module
```

Comment:
Do not put this in `test/conc/` either in `test/sym/contract/` or in `test/weasl/` for instance (For WEbAssembly Specification Language hahaha) but if you have any other idea, go with it, it's your tool :-)